tldw_Server_API/app/core/MCP_unified/tests/conftest.py

```
@@ -20,27 +21,19 @@ def mcp_ws_client(monkeypatch):
         server.config.ws_auth_required = False
         server.config.allowed_client_ips = []
         server.config.blocked_client_ips = []
-    except Exception:
-        # If server not yet initialized in tests, proceed; WS paths may init lazily
-        _ = None
-
-    client = TestClient(app)
-    try:
+        try:
+            server.config.debug_mode = True
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)